winui/Scheduler/Themes.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Themes in WinUI Scheduler control | Syncfusion
```

Comment:
title: Themes in WinUI Scheduler Control | Syncfusion

Reply:
Changed.